README.md

```diff
@@ -37,30 +37,56 @@ dumps
 └── users
     └── index
         └── 0.json
+        └── 0.request_body
+        └── 0.response_body
 ```
 
-The content of each dump file is a JSON object with attributes that contain
+The content of each `#.json` dump file is a JSON object with attributes that contain
 information about the HTTP request and response. For example:
 
 ```json
 {
   "request": {
     "method": "GET",
-    "url": "http://www.example.com/test",
-    "body": ""
+    "url": "http://www.example.com/test"
   },
   "response": {
     "status": 200,
     "statusText": "OK",
     "headers": {
       "Content-Type": "text/html; charset=utf-8",
       ...
-    },
-    "body": "<p>Hello World!</p>"
+    }
   }
 }
 ```
 
+The content of `#.request_body` and `#.response_body` are simply the request and
+response body contents. For example:
+
+`#.request_body` :
+```
+foo[bar]=baz
+```
+`#.response_body` :
+```
+<p>Hello World!</p>
+```
+Or, for a "multipart/formdata" request which contains a submitted file:  
+
+`#.request_body` :
+```
+------------XnJLe9ZIbbGUYtzPQJ16u1
+content-disposition: form-data; name="uploaded_image_file"; filename="fake_image_file.png"
+content-type: image/png
+content-length: 21
+
+<file data content appears here>
+------------XnJLe9ZIbbGUYtzPQJ16u1--
+```
+> Note: request/response bodies are held in separate files because multipart forms
+> may contain binaries which cannot be safely passed through the JSON formatter.
```

Comment:
This isn't a quote, so drop the `>`.

I also don't think it requires the "Note:" prefix.

````suggestion
```

Request and response bodies are held in separate files because multipart forms
may contain binary data which cannot be safely passed through the JSON formatter.
````

Reply:
I was formatting this like an "optional reading" block, but did not want to leave it out entirely. I'm happy to have it in the main text stream, though, so I'll implement this suggestion :+1: